test/credentials.spec.js

```diff
@@ -554,6 +566,7 @@
           expect(creds.secretAccessKey).to.equal('SECRET');
           expect(creds.sessionToken).to.equal('TOKEN');
           expect(creds.expireTime).to.eql(new Date(0));
+          delete process.env.AWS_SDK_LOAD_CONFIG;
```

Comment:
Is this delete necessary? Couldn't you restore `process.env` before each test, instead of before all?